templates/zerver/integrations/nagios.md

```diff
@@ -1,64 +1,60 @@
-{!create-stream.md!}
+1.  {!create-stream.md!}
 
-{!download-python-bindings.md!}
+1.  {!download-python-bindings.md!}
 
-Next, on your {{ settings_html|safe }}, create a bot for
-{{ integration_display_name }}.
+1.  Next, on your {{ settings_html|safe }}, create a bot for
+    {{ integration_display_name }}.
 
-Next, open `integrations/nagios/zuliprc.example` in your favorite
-editor, and change the following lines to specify the email address
-and API key for your Nagios bot, saving it to `/etc/nagios3/zuliprc`
-on your Nagios server:
+1.  Next, open `integrations/nagios/zuliprc.example` in your favorite
+    editor, and change the following lines to specify the email address
+    and API key for your Nagios bot, saving it to `/etc/nagios3/zuliprc`
+    on your Nagios server:
 
-```
-[api]
-email = NAGIOS_BOT_EMAIL_ADDRESS
-key = NAGIOS_BOT_API_KEY
-site = {{ api_url }}
-```
+        [api]
+        email = NAGIOS_BOT_EMAIL_ADDRESS
+        key = NAGIOS_BOT_API_KEY
+        site = {{ api_url }}
 
-Copy `integrations/nagios/zulip_nagios.cfg` to `/etc/nagios3/conf.d`
-on your Nagios server.
+1.  Copy `integrations/nagios/zulip_nagios.cfg` to `/etc/nagios3/conf.d`
+    on your Nagios server.
 
-Finally, add `zulip` to the `members` list for one or more of the
-contact groups in the `CONTACT GROUPS` section of
-`/etc/nagios3/conf.d/contacts.cfg`, doing something like:
+1.  Finally, add `zulip` to the `members` list for one or more of the
+    contact groups in the `CONTACT GROUPS` section of
+    `/etc/nagios3/conf.d/contacts.cfg`, doing something like:
 
-```
-define contactgroup {
-    contactgroup_name       admins
-    alias                   Nagios Administrators
-    members                 monitoring, zulip
-}
-```
+        define contactgroup {
+            contactgroup_name       admins
+            alias                   Nagios Administrators
+            members                 monitoring, zulip
+        }
```

Comment:
While this indentation does work, you could also just have intended the backticks-based code blocks 3 spaces so they fit within the bullet.  I kinda prefer the backticks style since it's possible to enable syntax highlighting with that style by just adding a word (they're otherwise equivalent).  Would you be up for redoing these to not remove the triple-backticks style of code blocks?

Reply:
Right okay, can update these, for the ones that are surrounded with { raw } in a couple of the docs, they had to be indented several tabs as otherwise the docs wouldn't render for some reason, not sure if there is some underlying issue with those ones elsewhere. @timabbott